emission/core/wrapper/trip.py

```diff
@@ -88,7 +114,15 @@ def get_distance(self):
         return cm.calDistance(self.trip_start_location, self.trip_end_location, True)
 
     def save_to_db(self):
-        pass
+        db = edb.get_trip_db()
+        for section in self.sections:
+            section.save_to_db()
+        db.insert({"user_id": self.user_id, "trip_id": self.trip_id, "sections": range(len(self.sections)), "trip_start_datetime": self.start_time,
```

Comment:
I think we do this to save the fake trips to the trip database and save the corresponding sections to the section database so that we can use them for testing.